tests/dao_tests/test_questionnaire_response_dao.py

```diff
@@ -847,6 +868,28 @@ def test_february_cope_survey(self):
         self.assertEqual(QuestionnaireStatus.SUBMITTED, participant_summary.questionnaireOnCopeFeb)
         self.assertEqual(num_completed_ppi_after_setup + 1, participant_summary.numCompletedPPIModules)
 
+    def test_cope_first_minute_survey(self):
+        """Make sure the dao fills in the summary data for the first COPE minute survey"""
+        self.insert_codes()
+        participant = self.data_generator.create_database_participant(participantId=1, biobankId=2)
+        self._setup_participant()
+
+        questionnaire = self._create_questionnaire(module_code=self.vaccine_1_survey_module_code)
+        authored_date = datetime.datetime(2021, 3, 4)
+        self.submit_questionnaire_response(
+            participant_id=to_client_participant_id(participant.participantId),
+            questionnaire_id=questionnaire.questionnaireId,
+            authored_datetime=authored_date
+        )
+
+        # Check that the summary fields have the survey data
+        summary: ParticipantSummary = self.session.query(ParticipantSummary).filter(
+            ParticipantSummary.participantId == participant.participantId
+        ).one()
+        self.assertEqual(QuestionnaireStatus.SUBMITTED, summary.questionnaireOnVaccineMinute1)
+        self.assertEqual(authored_date, summary.questionnaireOnVaccineMinute1Authored)
+
```

Comment:
Optional: There's an opportunity to use the PDRGeneratorTestMixin to validate we've also captured the questionnaire response details in the pdr_participant (nested module) data and/or the pdr_mod_cope_vaccine1 data.  (I can also look at adding it in a separate PDR)

Reply:
The PDR code would need to be updated first, right? I think that's the scope for DA-2043.

Reply:
For the participant data, if the Questionnaire/QuestionnaireResponse data exists in the test database then I think the .  participant data generators should pick it up in the participant's nested module data.  

I also already added skeleton support for the new module  as part of [PR #2452](https://github.com/all-of-us/raw-data-repository/pull/2452/filesl), for the BQPDRQuestionnaireResponseGenerator (we don't have a resource generator yet for the module response data).  This was when we thought the launch was going to be 6/3.  I may need to update the DA ticket.

I can make the unit test changes later.

Reply:
Ah, sorry. I forgot that was added.

Thanks for pointing out (and setting up) the PDRGeneratorTestMixin. I'll update the unit test.

Reply:
Ok, should be added. Let me know if there's anything that should be changed for how I tested.